### PR TITLE
Add traceability diagrams and guidance page

### PR DESCRIPTION
**Pull request by amy-silcock**
These diagrams are to be included in the new traceability guidance page.

### CODE REVIEW

File: en/traceability.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  
+  Each organisation involved in the activity (funding, accountable, extending or implementing) should be listed. 
+
+  The level of detail for each participating organisation will depend on where in the network they are or also any confidentiality concerns. You should provide as much information as possible from the full list below that is available to you at the time of publishing:
+
+  - Name
+  - Organisation `type <https://iatistandard.org/en/iati-standard/203/codelists/organisationtype/>`__ 
+  - Organisation `role <https://iatistandard.org/en/iati-standard/203/codelists/organisationrole/>`__
+  - IATI Organisation Identifier (if known)
+  - Unique IATI activity Identifier (if known)
```
Comment (AudreyIATI) [inline en/traceability.rst:56]: Inconsistent capitalisation - "activity" here needs capital A

File: en/traceability.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  Use your list of participating organisations to add details of the `provider-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/transaction/provider-org/>`__ and `receiver-org <http://iatistandard.org/203/activity-standard/iati-activities/iati-activity/transaction/receiver-org/>`__ for each transaction. For incoming transactions, there should be an external provider and the reporting-org is the receiver. For outgoing transactions, the reporting-org is the provider and an external organisation is the receiver-org.
+
+  You should provide as much of the information listed below as possible that is available to you at the time of publishing:
+
+  - Name
+  - Organisation `type <https://iatistandard.org/en/iati-standard/202/codelists/organisationtype/>`__ 
+  - IATI Organisation Identifier @ref (if known)
+  - Unique IATI activity identifier @provider-activity-id or @receiver-activity-id (if known)
+
```
Comment (AudreyIATI) [inline en/traceability.rst:75]: As above, should be Activity Identifier

**Review by AudreyIATI — COMMENTED** [review]
Reads really well and the visual of the diagram really helps to clarify who in the delivery chain needs to report what and when. No content changes from me, just minor syntax

**Review by AudreyIATI — APPROVED** [review]

File: en/traceability.rst (hunk 1 above)
Comment (amy-silcock) [reply en/traceability.rst]: I have changed :)

File: en/traceability.rst (hunk 2 above)
Comment (amy-silcock) [reply en/traceability.rst:75]: I have changed :)

**Review by andreaszenasidi — APPROVED** [review]
Reviewed and looks good to me.

**Comment by akmiller01** [comment]
@amy-silcock have we done embedded images in the guidance pages before? I'm getting this while attempting to render:
```
IATI-Guidance/en/traceability.rst:: WARNING: image file not readable: media/traceability_diagram_1.png
```

**Comment by amy-silcock** [comment]
Hi @akmiller01 we've done embedded images once before in the 'understanding results' piece: https://github.com/IATI/IATI-Guidance/blob/master/en/understanding-results.rst

I tried to keep everything the same, not sure on the image sizing though. That was a guess.

**Comment by akmiller01** [comment]
Gotcha, in this case it's a matter of case mismatch. The files are `media/traceability_diagram_1.PNG` and the RST is calling for `media/traceability_diagram_1.png`

**Review by nosvalds — APPROVED** [review]

**Comment by amy-silcock** [comment]
Please can we pull through to check on dev? The links should now be working

**Comment by amy-silcock** [comment]
Confirmed that links (spot checked) are working. Good to go live

**Pull request merged**